p2p/src/authenticated/actors/dialer.rs

```diff
@@ -24,7 +24,7 @@ use tracing::debug;
 
 pub struct Config<C: Scheme> {
     pub registry: Arc<Mutex<Registry>>,
-    pub connection: ConnectionConfig<C>,
+    pub stream_cfg: StreamConfig<C>,
```

Comment:
nit: I've been calling configs that represent some object (in case this `stream`) just that rather than `stream_cfg` (it is inferred from the type that it isn't a `stream`).

Find this makes things look a bit less verbose.